lib/widgets/selection.dart

```diff
@@ -743,13 +743,13 @@ class SelectionCheckmark extends StatefulWidget {
   _SelectionCheckmarkState createState() => _SelectionCheckmarkState();
 }
 
-class _SelectionCheckmarkState extends State<SelectionCheckmark> {
-  String _flareAnimation = 'stop';
+class _SelectionCheckmarkState extends State<SelectionCheckmark> with TickerProviderStateMixin {
```

Comment:
SingleTickerProviderStateMixin

Reply:
Cool, I didn't know about that one before.